src/openms/source/FORMAT/QcMLFile.cpp

```diff
@@ -1038,8 +1049,1023 @@ namespace OpenMS
     }
   }
 
-  void QcMLFile::store(const String& filename) const
+  double getMassDifference(double theo_mz, double exp_mz, bool use_ppm)
```

Comment:
we have some methods in place. I think somewhere in the Math namespace.
Can you check and get rid of the duplication?

Reply:
since this function was used only once in the code i removed it and did this simple calculation directly